### PR TITLE
Allow configuration of JWT inclusions via environment

### PR DESCRIPTION
**Pull request by davidbanham**

### CODE REVIEW

File: config/index.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     {"var": "API_KEY", "default": "hai"},
     {"var": "RETHINK_HOST", "default": "127.0.0.1"},
     {"var": "RETHINK_PORT", "default": "28015"},
-    {"var": "RETHINK_NAME", "default": "auth"}
+    {"var": "RETHINK_NAME", "default": "auth"},
+    {"var": "JWT_INCLUSIONS", "default": "email,id"}
```
Comment (nwinch) [inline config/index.json]: Maybe make that an array?  Then no need to split it further down.

Comment (davidbanham) [inline config/index.json]: Environment variables are just strings. No datatypes at all.

Comment (nwinch) [inline config/index.json]: Ah yes, you are correct.

**Comment by nwinch** [comment]
LGTM 👍

**Comment by nwinch** [comment]
@davidbanham Tests seem broken, but otherwise good to go.

**Comment by davidbanham** [comment]
It's just that flaky email uniqueness test. Definitely needs fixing, but unrelated to this pr.

**Comment by davidbanham** [comment]
Pull requests that have a failing status can’t be merged from a phone.

Ugh. Can someone hit merge?

**Comment by nwinch** [comment]
I will.  And good morning! :)

**Pull request merged**